x/cronos/types/types.go

```diff
@@ -1,8 +1,17 @@
 package types
 
-import "math/big"
+import (
+	"math/big"
+	"strings"
+)
 
 var (
 	Ten       = big.NewInt(10)
 	TenPowTen = Ten.Exp(Ten, Ten, nil)
 )
+
+// IsValidDenomToWrap returns if it's ok to wrap the native denom in erc20
+// Currently only supports ibc/{hash} and gravity{contract}
+func IsValidDenomToWrap(denom string) bool {
+	return strings.HasPrefix(denom, "ibc/") || strings.HasPrefix(denom, "gravity0x")
```

Comment:
types/params.go already have 
`validateIsIbcDenom` 

I think we will also need `validateIsGravityDenom` in the future (when we will add the mapping in genesis)

Maybe we could refactor this method to use both validate

Reply:
done